codersdk/workspaces.go

```diff
@@ -59,6 +65,8 @@ type CreateWorkspaceBuildRequest struct {
 	// This will not delete old params not included in this list.
 	ParameterValues     []CreateParameterRequest  `json:"parameter_values,omitempty"`
 	RichParameterValues []WorkspaceBuildParameter `json:"rich_parameter_values,omitempty"`
+
+	LogLevel ProvisionerLogLevel `json:"log_level,omitempty" validate:"omitempty,oneof=debug"`
```

Comment:
suggestion: Add a comment so this gets documented in the API doc

Reply:
Fixed!